Rectangle/ja.lproj/Main.strings

```diff
@@ -432,22 +432,22 @@
 "jBQ-r6-VK2.title" = "カーン";
 
 /* Class = "NSMenuItem"; title = "\tRight to Left"; ObjectID = "jFq-tB-4Kx"; */
-"jFq-tB-4Kx.title" = "\t右から左へ";
+"jFq-tB-4Kx.title" = "\t右から左";
 
 /* Class = "NSMenuItem"; title = "Format"; ObjectID = "jxT-CU-nIS"; */
 "jxT-CU-nIS.title" = "フォーマット";
 
 /* Class = "NSMenuItem"; title = "About"; ObjectID = "jxe-nr-LDQ"; */
-"jxe-nr-LDQ.title" = "アバウト";
+"jxe-nr-LDQ.title" = "Rectangleについて";
 
 /* Class = "NSMenuItem"; title = "Show Sidebar"; ObjectID = "kIP-vf-haE"; */
 "kIP-vf-haE.title" = "サイドバーを表示";
 
 /* Class = "NSTextFieldCell"; title = "Go to System Preferences → Security & Privacy → Privacy → Accessibility"; ObjectID = "lgE-cR-cQ5"; */
-"lgE-cR-cQ5.title" = "システム環境設定→セキュリティとプライバシー→プライバシー→アクセシビリティに移動します";
+"lgE-cR-cQ5.title" = "システム環境設定ー→セキュリティとプライバシー→プライバシー→アクセシビリティに移動してください";
 
 /* Class = "NSTextFieldCell"; title = "When the menu bar icon is hidden, relaunch Rectangle from Finder to open the menu."; ObjectID = "ltc-mf-BHr"; */
-"ltc-mf-BHr.title" = "メニューバーのアイコンが非表示になったら、FinderからRectangleを再起動してメニューを開きます。";
+"ltc-mf-BHr.title" = "メニューバーのアイコンが非表示になったら、FinderからRectangleを再起動してメニューを開く。";
```

Comment:
```suggestion
"ltc-mf-BHr.title" = "メニューバーのアイコンが非表示の場合にメニューを開くには、FinderからRectangleを再起動してください。";
```

I guess the "when" here means "in the case…", right?

Reply:
Ah, yes—this looks like a Settings entry, so that's how I interpreted it as well.

> Also, I thought it would be better to leave the conjugation in "dictionary form." Was that a good decision?

「 辞書形」にそのまま残した方がいいと思ったんだね。それでいい？

Reply:
> Also, I thought it would be better to leave the conjugation in "dictionary form." Was that a good decision?

The original English sentence sounds describing/encouraging how you can open the menu (tell me if it's not), so I think it should be imperative form instead, and plus adding politeness, so my suggestion is "再起動してください". Making it honorific would be necessary because it would sound rude if you're said "再起動しろ" 😅

Reply:
@ryonakano Ahh, I see what you mean now! I was reading it completely differently. Oops 😆 definitely gonna make that change!